go-controller/pkg/ovn/policy.go

```diff
@@ -1174,6 +1204,23 @@ func (oc *Controller) addNetworkPolicy(policy *knet.NetworkPolicy) error {
 
 	if len(nsInfo.networkPolicies) == 0 {
 		err = oc.createDefaultDenyPGAndACLs(policy.Namespace, policy.Name, nsInfo)
+		defer func() {
```

Comment:
so I think you can put this after line 1228. Since the last thing createDefaultDenyPGAndACLs does is the single transaction to create the default acls, if that fails no config is applied. So if you get an error here, we know no configuration has been applied in OVN so there is no reason to try to rollback.

Reply:
ack, yep makes sense.